packages/js/email-editor/src/components/preview/send-preview-email.tsx

```diff
@@ -119,10 +130,7 @@ function RawSendPreviewEmail() {
 									link: (
 										// eslint-disable-next-line jsx-a11y/anchor-has-content, jsx-a11y/control-has-associated-label
 										<a
-											href={ new URL(
-												'free-plan',
-												'https://www.mailpoet.com/'
-											).toString() }
+											href={ `https://account.mailpoet.com/?s=1&g=1&utm_source=mailpoet_email_editor&utm_medium=plugin&utm_source_platform=${ editorCurrentPostType }` }
```

Comment:
I know it is not part of this PR, but I think the modal window for sending the preview should not contain a URL defined by MailPoet.  Do you agree? If yes we should create a ticket to address it.

Reply:
Hmmm, this is more of a funnel to drive traffic to MailPoet and increase subscription sales. Basically, it's an upsell. I don't see any problem with keeping it here since what MailPoet sells is guaranteed inbox delivery of emails.